src/nvim/ui.c

```diff
@@ -51,7 +51,7 @@
 static UI *uis[MAX_UI_COUNT];
 static bool ui_ext[kUIExtCount] = { 0 };
 static size_t ui_count = 0;
-static int row = 0, col = 0;
+static int _row = 0, _col = 0;
```

Comment:
Why are these using a leading `_` when everything else is using a trailing one?

Reply:
these are static variables.

Reply:
Is this convention documented anywhere?